src/amber/controller/error.cr

```diff
@@ -33,6 +27,14 @@ module Amber::Controller
       end
     end
 
+    private def internal_server_error_message
```

Comment:
I'm fine with moving this into a private method especially since this only gets called if there's an error. 

Personally I think we abstract code into methods too much though. In a lot of cases it's just as readable either way and slows down the code minutely every branch we add to the AST.